libs/pages/logs/environment/src/lib/ui/pod-logs/pod-logs.tsx

```diff
@@ -175,6 +179,17 @@ export function PodLogs({
         setFilter={setFilter}
         filter={filter}
       >
+        {showPreviousLogs === false && (
+          <button
+            type="button"
+            className="block py-1.5 bg-neutral-500 text-neutral-250 text-center text-sm font-medium"
+            onClick={() => setShowPreviousLogs?.(true)}
+            style={{ inlineSize: '100%' }}
```

Comment:
I think you can use `w-full` instead of it, no?

Reply:
No it's a surprising HTML limitation. `width` doesn't work
https://stackoverflow.com/questions/9699039/button-with-displayblock-not-stretched

Reply:
I have added a comment
https://github.com/Qovery/console/pull/922/commits/8596115b8b454ba638a52bc4b0f4bf5459396686

Reply:
ok I tested with `w-full` and I don't see the problem, maybe it does not appear on a Mac

https://www.loom.com/share/7f469ca064494fda8ce943bad47b8789

Reply:
https://github.com/Qovery/console/pull/922/commits/7d0f913720eabb91e022427abf0edd06cdc04829